### PR TITLE
fix(tutorials): final review MTA-1685d

### PR DESCRIPTION
**Pull request by ldecarvalho-doc**

### CODE REVIEW

File: tutorials/actility-lorawan-operator/index.mdx [hunk 1]
```diff
@@ -13,97 +13,93 @@ dates:
...
+  - Pick any color to identify this subscription
+  - Leave QoS 0
+  - Use `my-app/things/+/uplink` as the topic. This will instruct the IoT Hub to send you uplink messages from all devices
+  - Click **Subscribe**
 
-## Verify the we have done things right
+  <Lightbox src="scaleway-mqtt_webclient_2.png" alt="" />
 
-It is time to see messages flowing on IoT Hub:
+Your subscription now appears in the **Subscriptions** block and the messages from your LoRa devices appear in the **Messages** block.
```
Comment (RoRoJ) [inline tutorials/actility-lorawan-operator/index.mdx:79]: should LoRa be written like the previous times with the C symbol?

File: tutorials/configure-plex-s3/index.mdx [hunk 2]
```diff
@@ -1,104 +1,96 @@
...
-<Message type="note">
-
-In this example, the bucket is named `plexmediaserver`. Bucket names are unique and can only exist once.
-
-</Message>
-
 ## Installing the required software
 
-1. Update the apt package manager and the software already installed on the system:
+1. Connect to your instance using SSH.
```
Comment (RoRoJ) [inline tutorials/configure-plex-s3/index.mdx:37]:
```suggestion
1. Connect to your Instance using SSH.
```

File: tutorials/configure-plex-s3/index.mdx [hunk 3]
```diff
@@ -173,22 +165,30 @@ In this example, the bucket is named `plexmediaserver`. Bucket names are unique
...
+  
+  <Lightbox src="scaleway-plex-server-setup.png" alt="" />
 
-  Enter the required parameters and click **Next** to complete the setup.
+3. Enter the required parameters and click **Next** to complete the setup.
 
   <Message type="note">
 
-  Make sure the Checkbox `Allow me to access my media outside my home` is ticked to access your media files from everywhere.
+  Make sure the Checkbox `Allow me to access my media outside my home` is ticked so you access your media files from any device.
```
Comment (RoRoJ) [inline tutorials/configure-plex-s3/index.mdx:176]:
```suggestion
  Make sure the Checkbox `Allow me to access my media outside my home` is ticked so you can access your media files from any device.
```

File: tutorials/actility-lorawan-operator/index.mdx [hunk 4]
```diff
@@ -13,97 +13,93 @@ dates:
...
 
-<Message type="note">
+2. Click **Applications > Create** and choose MQTT.
 
-- The device name is not important for this tutorial.
-- Keep the default `Deny Insecure` connections setting to provide maximum security.
+3. Fill in the following parameters (unlisted settings can be anything you like):
 
-</Message>
+    - **Hostname**: Your IoT Hub Endpoint, followed by the MQTT port (ex: `iot.fr-par.scw.cloud:8883`).
```
Comment (bene2k1) [inline tutorials/actility-lorawan-operator/index.mdx:46]:
```suggestion
    - **Hostname**: Your IoT Hub endpoint, followed by the MQTT port (ex: `iot.fr-par.scw.cloud:8883`).
```

**Review by RoRoJ — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Pull request merged**